src/components/Timer/Timer.tsx

```diff
@@ -37,6 +39,9 @@ export const Timer = (props: TimerProps) => {
   const allReady = useAppSelector((state) => state.participants!.others.filter((p) => p.connected && p.role === "PARTICIPANT").every((participant) => participant.ready));
   const isModerator = useAppSelector((state) => state.participants?.self.role === "OWNER" || state.participants?.self.role === "MODERATOR");
 
+  // Better way to get boardId? Please help :)
+  const boardId = `${useAppSelector((state) => state.board.data?.id)}`;
```

Comment:
```suggestion
  const boardId = useAppSelector((state) => state.board.data!.id);
```

This will do the trick. The board data is always available for the timer component.

Reply:
Works great! :-)